cmd/consensus/main.go

```diff
@@ -143,8 +145,19 @@ func main() {
 			flags.StringVar(&accessAddress, "access-address", "", "the address of an access node")
 			flags.StringVar(&secureAccessNodeID, "secure-access-node-id", "", "the node ID of the secure access GRPC server")
 			flags.BoolVar(&insecureAccessAPI, "insecure-access-api", true, "required if insecure GRPC connection should be used")
+			flags.StringVar(&startupTimeString, "startup-time", cmd.NotSet, "specifies date and time (in ISO 8601 format) when the consensus participant enters the first view")
```

Comment:
Please also adding the time format in the comment here as well.

Also it's better to mention that if this startup time is a time in the past, then the node will start up right away.

Reply:
Updated doc to reflect that start-time is the data *after which* the first view may be entered. Therefore if the start-time is in the past, then the first view may be entered immediately